File: Java/KeyboardInput.java
import java.util.Scanner;

public class KeyboardInput {

	public static void main(String[] args) {
    // Scanner Object
    Scanner scanner = new Scanner(System.in);
    System.out.println("Enter Yor Input: ");

    // Read keyboard input
    String input = scanner.nextLine();

    // Print inserted value
    System.out.println("Inserted Value: " + input);
	}
}

File: C#/KeyboardInput.cs
using System;

public class KeyboardInput
{
    public static void Main(string[] args)
    {
        Console.Write("Keyboard input: ");
        
        // Read keynboard input
        string input = Console.ReadLine();
        
        // Print Inserted value
        Console.WriteLine("Inserted value: " + input);
    }
}